### PR TITLE
Move Sub Link Caret Outside

### LINKED ISSUE #55: subtitle arrows should be outside the box
similar to navigation links

### PR DESCRIPTION
**Pull request by ckohen**
Closes: #55

### CODE REVIEW

File: components/Navigation.tsx [hunk 1]
```diff
@@ -116,7 +116,9 @@ function NavigationSubLink({ href, children }: NavigationSubLinkProps) {
     <span className="flex items-center ml-4">
       <Link href={href}>
         <a className={classes}>
-          {router.asPath === href ? <Caret className="mr-1 w-2 h-2" /> : null}
+          {router.asPath === href ? (
+            <Caret className="ml--1 absolute -m-4 mr-1 w-2 h-2" />
```
Comment (IanMitchell) [inline components/Navigation.tsx]: think the `ml--1` can be removed, doesn't do anything (should it be `-ml-4`?

Comment (ckohen) [reply components/Navigation.tsx]: lol, I can't believe I left that in. Fixed that and added the relative to the parent so the absolute is in the right place 😄

File: components/Navigation.tsx [hunk 2]
```diff
@@ -113,10 +113,12 @@ function NavigationSubLink({ href, children }: NavigationSubLinkProps) {
   );
 
   return (
-    <span className="flex items-center ml-4">
+    <span className="relative flex items-center ml-4">
       <Link href={href}>
         <a className={classes}>
-          {router.asPath === href ? <Caret className="mr-1 w-2 h-2" /> : null}
+          {router.asPath === href ? (
+            <Caret className="absolute -ml-4 w-2 h-2" />
```
Comment (IanMitchell) [inline components/Navigation.tsx:120]: do you need the relative / absolute for this?

Comment (ckohen) [reply components/Navigation.tsx:120]: I do to not have to do magic margins

File: components/Navigation.tsx (hunk 1 above)
Comment (IanMitchell) [reply components/Navigation.tsx]: Just delete the ml--1 and then merge this sucker!

Comment (ckohen) [reply components/Navigation.tsx]: how in the WHAT is it still there

Comment (ckohen) [reply components/Navigation.tsx]: oh it isn't

**Pull request merged**